tests/models/test_feed.py

```
@@ -0,0 +1,47 @@
+
+    def test_feed_instantiation(self):
+        # Test basic instantiation with required fields
+        feed = Feed(title="Test Feed", url="http://example.com", genre="News")
+        self.assertEqual(feed.title, "Test Feed")
+        self.assertEqual(feed.url, "http://example.com")
+        self.assertEqual(feed.genre, "News")
+        self.assertEqual(feed.description, "")
+        self.assertIsNone(feed.deleted_at)
+
+        # Test instantiation with optional fields
+        feed = Feed(title="Test Feed", url="http://example.com", genre="News", description="A test feed", deleted_at=datetime(2023, 1, 1))
```

Ruff (0.8.0)

`datetime.datetime()` called without a `tzinfo` argument

(DTZ001)